bluejeans/src/bluejeans_create.js

```diff
@@ -24,6 +24,16 @@ async function _command(params = {}, commandText, secrets = {}) {
     };
   }
 
+  const startTime = new Date(start).getTime();
+  const endTime = new Date(end).getTime();
+
+  if (endTime <= startTime || startTime <= Date.now()) {
```

Comment:
I think we can let the start time be the same as the present time but not before. What do you all think?

Reply:
Did you test it out? How will the meeting invite look like? Is it going to show that the meeting has started already?

Reply:
That would panic the attendee. Haha. Let's keep it as is.